framework/auth/oauth_scopes.py

```diff
@@ -233,18 +236,18 @@ class ComposedScopes(object):
                         (CoreScopes.PREPRINT_FILE_WRITE,)
 
     # Privileges relating to who can access a node (via contributors or registrations)
-    NODE_ACCESS_READ = (CoreScopes.NODE_CONTRIBUTORS_READ, CoreScopes.NODE_REGISTRATIONS_READ,
+    NODE_ACCESS_READ = (CoreScopes.NODE_CONTRIBUTORS_READ, CoreScopes.NODE_REGISTRATIONS_READ, CoreScopes.SUBJECTS_READ,
                         CoreScopes.NODE_VIEW_ONLY_LINKS_READ, CoreScopes.REGISTRATION_VIEW_ONLY_LINKS_READ,
                         CoreScopes.NODE_REQUESTS_READ, CoreScopes.NODE_SETTINGS_READ)
     NODE_ACCESS_WRITE = NODE_ACCESS_READ + \
-                            (CoreScopes.NODE_CONTRIBUTORS_WRITE, CoreScopes.NODE_REGISTRATIONS_WRITE,
+                            (CoreScopes.NODE_CONTRIBUTORS_WRITE, CoreScopes.NODE_REGISTRATIONS_WRITE, CoreScopes.SUBJECTS_WRITE,
                              CoreScopes.NODE_VIEW_ONLY_LINKS_WRITE, CoreScopes.REGISTRATION_VIEW_ONLY_LINKS_WRITE,
                              CoreScopes.NODE_REQUESTS_WRITE, CoreScopes.NODE_SETTINGS_WRITE)
 
     # Privileges relating to who can access a preprint via contributors
-    PREPRINT_ACCESS_READ = (CoreScopes.PREPRINT_CONTRIBUTORS_READ,)
+    PREPRINT_ACCESS_READ = (CoreScopes.PREPRINT_CONTRIBUTORS_READ, CoreScopes.SUBJECTS_READ,)
     PREPRINT_ACCESS_WRITE = PREPRINT_ACCESS_READ + \
-                            (CoreScopes.PREPRINT_CONTRIBUTORS_WRITE,)
+                            (CoreScopes.PREPRINT_CONTRIBUTORS_WRITE, CoreScopes.SUBJECTS_WRITE,)
```

Comment:
hmm, do I need subjects read/write scopes.  Thinking no, now, because it's now making me think people can add subjects, which they can't. scopes for updating nodes/registrations/preprints/collection submissions probably sufficient.